CONTRIBUTING_TO_OPENSHIFT_JENKINS_IMAGE_AND_PLUGINS.md

```diff
@@ -0,0 +1,531 @@
+
+### Step 1: updating via "api.ci", the OpenShift Prow CI system
+
+As referred to previously, the new plugin version will land at `https://updates.jenkins.io/download/plugins/<plugin-name>`.  Monitor that page for the existence of the new version of the plugin.  Warning: the link for the new version can show up, but does not mean the file is available yet.  Click the link to confirm you can download the new version of the plugin.  When you can download the new version file, the new release is available.
+
+At this point, we are back to the steps articulated in [the base plugin installation section of this repository's README](https://github.com/openshift/jenkins/blob/master/README.md#base-set-of-plugins).  You'll 
+modify the [text file](https://github.com/openshift/jenkins/blob/master/2/contrib/openshift/base-plugins.txt) with the new version for whatever OpenShift plugin you have cut a new version for, and create a new PR.  The image build and e2e extended test runs we detailed in ["Actual PR Testing"](#actual-pr-testing) will commence.
+
+If the PR passes all tests and merges, the api.ci system will promote the jenkins images to quay.io for 4.x, and we have the separate jenkins job in the openshift ci jenkins server to push the 3.11 images to docker.io.
+
+The image build in particular is of interest when it comes to plugin versions within our openshift/jenkins image, and what we have to do in creating the RPM based images hosted on registy.redhat.io/registry.access.redhat.com that we provide subscription level support for.  The PR will have a link to the `ci/prow/images` job.  If you click that link, then the `artifacts` link, then the next `artifacts` link, then `build-logs`, you'll see gzipped output from each of the image builds.  Click the one for the main jenkins image.  If you search for the string `Installed plugins:` you'll find the complete list of every plugin that was installed.  Copy that output to clipboard and paste it into the PR that just merged.  See [https://github.com/openshift/jenkins/pull/829#issuecomment-477637521](https://github.com/openshift/jenkins/pull/829#issuecomment-477637521) as an example.
+
+You are now ready for the next step
+
+### Step 2: updating OSBS/Brew for generating the officially supported images available with Red Hat subscriptions
+
+First, some background:  for all Red Hat officially supported content, to ensure protection from outside hackers, all content is built in a quarantined system with no access to the external Internet.  As such, we have to inject all content into OpenShift's Brew server (see links like [https://docs.brew.sh/Homebrew-on-Linux](https://docs.brew.sh/Homebrew-on-Linux) if you are interested in the details/histories of this Linux build infrastructure), which is then scrubbed before official builds are run with it.  The injection is specifically the creation of an RPM which contains all the plugin binaries.
```

Comment:
AIUI the "Brew" build server here doesn't have anything to with "homebrew", which is a package manager for MacOS (and, according to the link here, also Linux).

"Brew" in the case of this build server is instead just the name of an instance of "Koji", so if there's going to be a link here, I'd suggest these two (Koji and OSBS):

https://pagure.io/koji/
https://osbs.readthedocs.io/en/latest/

Reply:
cool ! ... thanks for the education @grdryn - updates will be part of the next push